pages/api/account/manage/events.js

```diff
@@ -49,6 +51,37 @@ export async function getEventsApi(username) {
   } catch (e) {
     log.error(e, `failed to get events for username: ${username}`);
   }
+  if (getEvents.length > 0) {
+    let dateEvents = [];
+    const today = new Date();
+    getEvents.map((event) => {
+      let cleanEvent = JSON.parse(JSON.stringify(event));
+      try {
+        const start = new Date(event.date.start);
+        const end = new Date(event.date.end);
+        cleanEvent.date.startFmt = dateFormat({
+          format: "long",
+          date: event.date.start,
+        });
+        cleanEvent.date.endFmt = dateFormat({
+          format: "long",
+          date: event.date.end,
+        });
+
+        cleanEvent.date.cfpOpen =
+          event.date.cfpClose && new Date(event.date.cfpClose) > today;
+        cleanEvent.date.future = start > today;
+        cleanEvent.date.ongoing = start < today && end > today;
+        dateEvents.push(cleanEvent);
+      } catch (e) {
+        log.error(e, `ERROR event date for: "${event.name}"`);
+      }
+    });
+
+    getEvents = dateEvents;
+  } else {
+    getEvents = [];
+  }
```

Comment:
Is the `else` needed as `getEvents` is defined with `[]`?

Reply:
ooh let me remove the else statement.